docs/topics/about-ide-addons.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+// * docs/intellij-idea-plugin-guide/master.adoc
+// * docs/eclipse-code-ready-guide/master.adoc
+
+[id="about-ide-addons_{context}"]
+= About the {ProductShortName} {AddonType} for {IDEName}
+
+Migrate and modernize applications by using the {ProductName} ({ProductShortName}) {AddonType} for {IDEName}.
```

Comment:
I think the imperative voice sounds jarring, like an ad, when the rest of the module is declarative. Suggest changing to "You can migrate and modernize your applications ..."

Just a suggestion.